client/src/lib/components/sandbox/arena/ArenaCanvasShootingPhase.svelte

```diff
@@ -220,6 +225,16 @@
     await rerender();
   }
 
+  const rollDiceForAttack = async(rangedAttack: RangedAttackAction) => {
+    const attackingPieceId = rangedAttack.gamePieceId;
+    const attackingPiece = Utils.gamePieceById(attackingPieceId, gamePieces);
+    const attackingUnit = attackingPiece?.playerUnit.unit;
+    const numAttacks = attackingUnit?.rangedNumAttacks || 1;
+    const numRolls = numAttacks * 3;
+    console.log(`Rolling for ${numAttacks} attacks from Piece ${attackingPiece?.playerUnit.name} (ID: ${attackingPieceId})`);
+    return await new DiceRollServiceClient().getDiceRolls(numRolls, 6);
```

Comment:
Something about this will need to be updated eventually, either in the dice roll service, or in the client.  The problem is there is a single ciphertext here which maps to multiple rolls (times multiple numbers of attacks).  The proof needs to verify the ciphertext, but can only verify one attack at a time.  I think it was written to expect exactly 3 rolls, and to be called once per attack.

I am working on putting that in the backend server next, so we can leave this client implementation until the server has the requisite parts to execute it.